### PR TITLE
Feature/testing

### LINKED ISSUE #29: Testing
- [ ] build test for RecipeRepository
- [ ] Use smaller, sample data files for tests
- [ ] include sad paths (4 on rubric)

### PR DESCRIPTION
**Pull request by rhen92**
# Description of Changes

- Finish RecipeRepository test
- Made smaller sample data file
- Cleaned up tests 
- Cleaned up classes

# Linked Issues

- Closes #29

### CODE REVIEW

File: src/RecipeRepository.js [hunk 1]
```diff
@@ -3,21 +3,21 @@ class RecipeRepository {
     this.recipeList = recipeData;
   }
 
-  filterRecipesByTag() {
-    return this.recipeList.filter(recipe => recipe.tags.includes(tag));
+  filterRecipesByTag(keyword) {
```
Comment (battan40) [inline src/RecipeRepository.js:6]: This was a good save of the work that was already done

**Review by battan40 — APPROVED** [review]

File: src/RecipeRepository.js [hunk 2]
```diff
@@ -3,21 +3,21 @@ class RecipeRepository {
...
+    return this.recipeList.filter(recipe => recipe.tags.includes(keyword));
   }
 
   filterRecipesByName(keyword) {
-    const nameList = this.recipeList.map(rec => rec.name).map(title => title.split(' ')).flat();
+    const nameList = this.recipeList.map(rec => rec.name);
     const filteredNameList = nameList.filter(name => name.toLowerCase().includes(keyword.toLowerCase()));
-    return this.recipeList.filter(recipe => recipe.name.split(' ').some(word => filteredNameList.includes(word)))
+    return this.recipeList.filter(recipe => recipe.name.split().some(word => filteredNameList.includes(word)));
```
Comment (mistercanderson) [inline src/RecipeRepository.js:13]: this was a nice cleanup of the original method

File: src/test-data/sample-ingredient-data.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    "estimatedCostInCents": 617
+  },
+  {
+    "id": 2050,
+    "name": "vanilla",
+    "estimatedCostInCents": 926
+  }
+]
+
+export default ingredientsData;
```
Comment (mistercanderson) [inline src/test-data/sample-ingredient-data.js:59]: Way to cut down on the unnecessary amount of fake data needed for the tests here

File: test/recipe-test.js [hunk 4]
```diff
@@ -1,45 +1,43 @@
...
     expect(recipe.id).to.eq(595736);
   });
 
-  it('should initialize with an name', function() {
+  it('should initialize with an name', () => {
     expect(recipe.name).to.eq('Loaded Chocolate Chip Pudding Cookie Cups');
   });
 
-  it('should initialize with an image', function() {
+  it('should initialize with an image', () => {
```
Comment (mistercanderson) [inline test/recipe-test.js:32]: Welcome to the future it's ES6 time lets gooo

**Review by mistercanderson — APPROVED** [review]

**Pull request merged**